.gitignore

```diff
@@ -5,3 +5,4 @@
 distro/
 pmm-admin
 pmm.yml
+.idea/
```

Comment:
Put it before `.vscode/` for consistency. And always add line feed at the end. It is in Settings->Editor->General->Other->Ensure line feed at file end on save or something like that.

Reply:
Thank you, fixed